File: apiEZPZ/src/main/java/com/example/apiezpz/checklist/dto/ChecklistDTO.java
package com.example.apiezpz.checklist.dto;

import lombok.*;

import java.time.LocalDate;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ChecklistDTO {
    private Long id;
    private String title;
    private LocalDate departureDate;
    private LocalDate returnDate;
    private List<CategoryDTO> categories;   //카테고리 목록
}

File: apiEZPZ/src/main/kotlin/com/example/apiezpz/checklist/dto/ChecklistDTO.kt
package com.example.apiezpz.checklist.dto

import java.time.LocalDate

data class ChecklistDTO(
    var id: Long? = null,
    var title: String = "",
    var departureDate: LocalDate? = null,
    var returnDate: LocalDate? = null,
    var categories: List<CategoryDTO> = emptyList()   // 카테고리 목록
) 